recipe/meta.yaml

```diff
@@ -5,10 +5,11 @@ package:
   version: {{ version }}
 
 source:
-  fn: notebook-{{ version }}.tar.gz
+  fn: {{ version }}.tar.gz
```

Comment:
This would collide with other names if we aren't careful (is that why you had a SHA issue in the first place?)